### DISCUSSION TITLE
Welcome to xcalibrate Discussions!

### DISCUSSION THREAD

**Discussion by reinderien** [reinderien/xcalibrate#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jgcramer** [comment]
Hello,

Have used your application over xinput .. its did a good calibration. But after a reboot the settings are gone.

Any idea?

**Reply by Kippykip** [reply]
By memory I think you had to open a text editor with root and copy the values on screen to this file to make it permanent:
'/usr/share/X11/xorg.conf.d/99-libinput-ts-calib.conf'
You could also use a different name too I think, as long as it started with **99-**

Alternatively I made this fork which already saves it and makes it permanent anyway (but you have to run the script as root, so **sudo ./xcalibrate**). The fork also addresses the bug where some devices report the touchscreen as a keyboard and didn't show the device on the list such as the Panasonic Toughbook series of laptops. 
https://github.com/Kippykip/xcalibrate

**Comment by jgcramer** [comment]
Tried your new application, but when i run it its giving an error on line 1

Op do 12 mei 2022 om 17:03 schreef Kippykip ***@***.***>:

**Comment by jgcramer** [comment]
werkt toch had hem denk verkeerd gedownload!

Op za 14 mei 2022 om 12:08 schreef Guy Cramer ***@***.***>:

**Reply by reinderien** [reply]
Bedankt dat je het hebt uitgeprobeerd. Succes.

**Comment by antonmusalov** [comment]
Hey there, can I buy you a coffee?

Cheers

**Reply by reinderien** [reply]
That's generous - thank you. Please read https://github.com/reinderien/xcalibrate/blob/main/funding.md

**Comment by 4f4vtj749f-code** [comment]
So, I am new to all things Linux.  I am using Linux Mint Cinnamon, a touchscreen monitor (Aspen Tech. ATM123ROB) with a usb touchscreen (idealTEK URTC1000).  I cannot calibrate the pointer  on the screen with xinput calibrator, I tried loading Xcalibrate but cannot get past error ‘unable to find package’ despite updating and upgrading in the terminal.  It is running X11, I did look into something about repositories which lost me a bit however,  I am very reluctant to go back to windows 7 and am very impressed with Linux Mint Cinnamon, the reason for old hardware I use is for touchscreen jukeboxes for fun and recycling.  Any thoughts/ suggestions will be much appreciated.

**Reply by 4f4vtj749f-code** [reply]
Zena

**Reply by reinderien** [reply]
Can you please show the full text of the error? It's likely that you're just missing Numpy.

**Reply by 4f4vtj749f-code** [reply]
I managed to get ChatGPT to help sort it out in the end.  It got me to cut n paste code into the terminal and feedback the results, it took ages but I was determined to get it to work. It was something to do with the start up file conflicting with Libinput.  There is no way I could have sussed it all out on my own. Thanks for replying to my request, much appreciated.  There’s a beer coming your way!

**Reply by 4f4vtj749f-code** [reply]
I made a donation with  the Indian Residential School Survivors Society. Reinderien / Xcalibrate saved my old touchscreen jukebox, I am indebted.
